libraries/AP_BattMonitor/AP_BattMonitor_Params.h

```diff
@@ -27,6 +27,7 @@ class AP_BattMonitor_Params {
         BattMonitor_TYPE_FuelLevel_PWM              = 12,
         BattMonitor_TYPE_SUI3                       = 13,
         BattMonitor_TYPE_SUI6                       = 14,
+        BattMonitor_TYPE_NeoDesign                  = 15
```

Comment:
You need to add the number for the driver to the param docs, in `AP_BattMonitor_Params.cpp`.